package.json

```diff
@@ -31,7 +31,7 @@
     "dependencies": {
         "comb": "~1.0.0",
         "comb-proxy": "~1.0.0",
-        "commander": ">=0.5.1",
+        "commander": "~2.9.0",
```

Comment:
well we just did another PR for problem with using >= on comb-proxy.  I don't think we ever want to have dep's with >= in them since that could potentially bring in a large change.